src/components/Image.js

```diff
@@ -109,16 +111,23 @@ export const Image = ({
         resizeMode={resizeMode}
         borderRadius={borderRadius}
       >
-        {!!message && <ImageMessage message={message} />}
-        {!!globalSettings?.showImageRights && !!sourceProp?.copyright && (
-          <ImageRights imageRights={sourceProp.copyright} />
-        )}
+        <View style={styles.contentContainerStyle}>
+          {!!message && <ImageMessage message={message} />}
+          {!!button && <ImageButton button={button} />}
+          {!!globalSettings?.showImageRights && !!sourceProp?.copyright && (
+            <ImageRights imageRights={sourceProp.copyright} />
+          )}
+        </View>
       </RNEImage>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  contentContainerStyle: {
+    height: '100%',
+    justifyContent: 'flex-end'
```

Comment:
does this work in any case of possible children components?
100% means full height of the image?

Reply:
This style works correctly for all children components. 
Yes, the height value is directly proportional to the image. Covering 100% of the image's height